kvdb-rocksdb/src/lib.rs

```diff
@@ -167,6 +167,12 @@ pub struct DatabaseConfig {
 	pub columns: u32,
 	/// Specify the maximum number of info/debug log files to be kept.
 	pub keep_log_file_num: i32,
+	/// Enable RocksDB statistics.
```

Comment:
```suggestion
	/// Enable native RocksDB statistics.
```

(we have some statistics which we collect on our own)